tide/master/MasterApplication.cpp

```diff
@@ -94,9 +94,9 @@ MasterApplication::MasterApplication(int& argc_, char** argv_,
     , _masterToForkerChannel(new MasterToForkerChannel(forkChannel))
     , _masterToWallChannel(new MasterToWallChannel(worldChannel))
     , _masterFromWallChannel(new MasterFromWallChannel(worldChannel))
+    , _lock(new ScreenLock)
```

Comment:
since ScreenLock derives from enable_shared_from_this, please follow the same logic as for Options and Markers to enforce the creation of the object via a static create() method that returns a shared_ptr